tier1/{{cookiecutter.project_slug}}/repolinter.json

```diff
@@ -103,9 +103,9 @@
         "rule": {
           "type": "file-existence",
           "options": {
-            "globsAny": [
-              "{docs/,.github/,}CODE_OF_CONDUCT.md"
-            ]
+            "globsAny": ["{docs/,.github/,}CODE_OF_CONDUCT.md"],
+            "file-name": "CODE_OF_CONDUCT.md",
+            "file-content": "# Code of Conduct \n"
```

Comment:
Change to: `Contributor Code of Conduct`

https://github.com/DSACMS/repo-scaffolder/blob/main/tier2/%7B%7Bcookiecutter.project_slug%7D%7D/CODE_OF_CONDUCT.md?plain=1#L1C3-L1C30